Cartfile.resolved

```diff
@@ -1,5 +1,4 @@
 github "BoltsFramework/Bolts-ObjC" "1.9.0"
 github "BoltsFramework/Bolts-Swift" "e7811181243e9dfa9462ec3048cd5957519fa26f"
-github "ParsePlatform/Parse-SDK-iOS-OSX" "1.17.1"
+github "ParsePlatform/Parse-SDK-iOS-OSX" "1.16.0"
```

Comment:
Why not revert to 1.17.0?

Reply:
Wasnt there an issue that affected queries with 1.17.0 or did that also exist in 1.16.0.

Reply:
I think you’re right. If you think 1.16.0 is the right move then that’s fine

Reply:
I think so. Just need your approval.